### ISSUE TITLE
Figure out star content and turn it on

### ISSUE THREAD

**Issue by GeorgeBellTMH** [themeetinghouse/mobile#345]
We need some star content...once it's done we can turn the menu item back on.

**Comment by GeorgeBellTMH** [comment]
Some ideas for initial star content:

- [x] make a difference (duplicate content on website)
- [x] spiritually curious (duplicate content from website)
- [x] about us (duplicate content from website)

**Comment by GarnetMason** [comment]
I agree with all three of these pieces for the initial star content section.
